docs/pages/cdp/cdp-Log-and-trace-files.md

```diff
@@ -60,6 +60,6 @@ They can all be useful when identifying the cause of an error, especially STDERR
 
 ### JES job logs
 
-If you are familiar with the z/OS environment, you can often find some useful diagnostics by accessing the JES job log associated with the CICS region you deployed to.  The `cics-deploy push` command outputs the job IDs for all the CICS regions in the scope after a deployment.  The MSGUSR, SYSPRINT, SYSOUT and JESMSGLG spool files can all provide useful information.   If you are not so fluent with the world of z/OS, you may need to discuss your needs with one of your organization's CICS system administrators.
+If you are familiar with the z/OS® environment, you can often find some useful diagnostics by accessing the JES job log associated with the CICS® region you deployed to.  The `cics-deploy push` command outputs the job IDs for all the CICS® regions in the scope after a deployment.  The MSGUSR, SYSPRINT, SYSOUT and JESMSGLG spool files can all provide useful information.   If you are not so fluent with the world of z/OS, you may need to discuss your needs with one of your organization's CICS® system administrators.
```

Comment:
Can you pls remove double and tripple spaces after full stops. I realise HTML will collapse them but they are unnecessary.